dags/mozaggregator_prerelease.py

```diff
@@ -44,3 +54,51 @@
     ),
     dag=dag,
 )
+
+task_id = "prerelease_telemetry_aggregate_view_dataproc"
+gcp_conn = GoogleCloudBaseHook("google_cloud_airflow_dataproc")
+keyfile = json.loads(gcp_conn.extras["extra__google_cloud_platform__keyfile_dict"])
+client_email = keyfile["client_email"]
+
+prerelease_telemetry_aggregate_view_dataproc = SubDagOperator(
+    task_id=task_id,
+    dag=dag,
+    subdag=moz_dataproc_pyspark_runner(
+        parent_dag_name=dag.dag_id,
+        dag_name=task_id,
+        job_name="prerelease aggregates",
```

Comment:
This will fail due to whitespace. See https://github.com/mozilla/telemetry-airflow/commit/96f6a96d3586414402f34a703537bf9043e26af2

I'm wondering if this could be checked during tests, maybe as some assertion in the operator code?

Reply:
I checked this with manual testing, and found a slew of other bugs. This should be fine now.